src/pad/src/pad.tcl

```diff
@@ -353,7 +354,7 @@ sta::define_hidden_cmd_args "make_fake_io_site" {-name name \
 proc make_fake_io_site {args} {
   sta::parse_key_args "make_fake_io_site" args \
     keys {-name -width -height} \
-    flags {}
+    flags {};# checker off
```

Comment:
This is a public command despite its odd name

Reply:
May I ask what does "public command" mean? And also why does it being public mean we are ignoring checks for it?

Reply:
By public I mean this is a command we expect some users to use (unlike say a debug command).  I'm wondering why we are ignoring checks for such a command.

Reply:
Oh I see. We are ignoring checks because it does not appear in `help`. We are only making sure the counts in `help`, `proc` and `README` match

Reply:
@gadfort is there a reason for make_fake_io_site not to be in help?  Its in the README

Reply:
I don't see why it should be excluded.

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/db6d999c3804c3c433e85bb7a5270826b1579f34/src/pad/src/pad.tcl#L349 shows sta::define_hidden_cmd_args is used

Reply:
Ahh, that is an oversight on my part. It should be define_cmd_args instead

Reply:
@luarss please fix this in this PR to avoid any merge conflicts.